torch/_subclasses/base_tensor.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+# Ideally, tensor subclasses would would inherit directly from Tensor.
+# This is just our staging ground for applying behavior that hasn't yet made it
+# into the core Tensor class but that we would like to apply by default.
+class BaseTensor(torch.Tensor):
+    # See https://github.com/pytorch/pytorch/pull/73727 ; this is necessary
+    # to ensure that super().__new__ can cooperate with each other
+    @staticmethod
+    def __new__(cls, elem, *, requires_grad=None):
+        if requires_grad is None:
+            return super().__new__(cls, elem)  # type: ignore
+        else:
+            return cls._make_subclass(cls, elem, requires_grad)
+
+    # To ensure constructors can cooperate with one another, must accept and
+    # ignore element tensor (TODO: is this right???)
+    def __init__(self, elem):
+        super().__init__()
```

Comment:
I'm pretty confident we can dump this constructor, it never actually got used for anything